corehq/apps/accounting/models.py

```diff
@@ -679,12 +679,9 @@ def new_rate(cls, feature_name, feature_type,
         feature, _ = Feature.objects.get_or_create(name=feature_name, feature_type=feature_type)
         rate = FeatureRate(feature=feature)
 
-        if monthly_fee is not None:
-            rate.monthly_fee = monthly_fee
-        if monthly_limit is not None:
-            rate.monthly_limit = monthly_limit
-        if per_excess_fee is not None:
-            rate.per_excess_fee = per_excess_fee
+        rate.monthly_fee = monthly_fee or rate.monthly_fee
```

Comment:
you noted that if any of these are falsey this might do the wrong thing (not assign when it should). I think this could be more trouble than it's worth